tools/builds/Dockerfile-openstack

```diff
@@ -18,10 +19,14 @@ RUN mkdir -p /etc/ansible \
 RUN pip3 install --no-cache --upgrade pip
 
 RUN pip3 install --no-cache \
-    ansible \
-    openstacksdk \
     python-heatclient \
     python-openstackclient \
+    python-neutronclient \
+    python-keystoneclient \
+    python-novaclient \
+    python-glanceclient \
+    ansible \
```

Comment:
Is this Dockerfile going to be used for actual deployments or just testing? If actual, we should pin to specific versions in pip. Some module names change for OpenStack between Ansible 2.9 and earlier.

Reply:
It's currently only used to deploy from the laptop.

I want us to start populating docker images, and have something mature for the day we need them.

Like mentioned by @wkulhanek we could use UBI here